server/routerlicious/packages/lambdas/src/deli/lambda.ts

```diff
@@ -425,6 +427,46 @@ export class DeliLambda extends TypedEventEmitter<IDeliLambdaEvents> implements
                         sequencedMessage.expHash1 = this.lastHash;
                     }
 
+                    if (this.serviceConfiguration.deli.enableAutoSummaryAcks &&
```

Comment:
i really just want to preserve a hook for the server to do something if it wants to, whereas this precludes that, and i don't think we'd ever want deli to do something blocking.

Reply:
i also don't want to totally bifurcate our service pieces, so i want to support just acking, and doing more via the same hook

Reply:
Scribe runs as its own kafka consumer group, so it listens / consumes every sequenced op. Having scribe run just to create summaryAck's would not be great - it would remove a major benefit of the single-commit work.
I wanted to stop running scribe, which means the server would run 1 less lambda / node process / kafka consumer group. So Push could stick with this logic (having it in deli). If FRS or someone else wanted extra logic, they could keep using scribe and update it.

Reply:
if i turned on this config, and had scribe running what would happen? my guess is thrashing and/or multiple acks. it just sees really easy to fat finger the wrong behavior. maybe deli should support inlining services that process message or something, and this could be one

Reply:
Scribe would see that deli acked the message and do nothing.

I could instead make deli emit an event when it gets a summarize message. Then it's up to the hosting service (FRS / Push) to handle that accordingly
Push would run the same logic in this if block. Scribe would still need the logic I'm adding to ignore the summarize messages though.

Reply:
something like that sounds better to me. i think its easier to manage composable pieces rather than a config matrix.

Reply:
Another option - I think I could also handle all of this in Push and not make any changes in deli/scribe, but that would make it harder for FRS to onboard to the single-commit logic

Reply:
I changed it to emit a message